adafruit_mcp2515/__init__.py

```diff
@@ -841,6 +831,11 @@ def listen(self, matches=None, *, timeout: float = 10):
         """
         if matches is None:
             matches = []
+        elif self.silent and not self.loopback:
+            raise AttributeError(
+                "Hardware does not support setting `matches` in when\
+                `silent`==`True` and `loopback` == `False`"
```

Comment:
interesting restriction.

Reply:
I think they imagine that if you're in the monitoring mode that you want to "hear" everything?